index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/lozad/dist/lozad.min.js"
+    ></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="navigation">
+          <div class="menu">
+            <div class="header">
+              <div id="icon">
+                <a>
```

Comment:
Why we got this `anchor` without a link?

Reply:
🤔 that's strange, i'll check that